ios/RNBranch.m

```diff
@@ -402,6 +402,17 @@ - (BranchUniversalObject *)findUniversalObjectWithIdent:(NSString *)ident reject
     resolve([self.class.branch getFirstReferringParams]);
 }
 
+#pragma mark lastAttributedTouchDataWithAttributionWindow
+RCT_EXPORT_METHOD(
+                  lastAttributedTouchDataWithAttributionWindow:(NSNumber* __nonnull)window
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejector:(__unused RCTPromiseRejectBlock)reject
+                  ) {
+    [self.class.branch lastAttributedTouchDataWithAttributionWindow:window.integerValue completion:^(BranchLastAttributedTouchData *r){
+        resolve(r);
```

Comment:
You have a promise rejection on Android but not here. Is it possible to get an error here?

Reply:
Correct me if I am wrong, but I think that would require a change to the native iOS SDK since https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/blob/master/Branch-SDK/Branch.m#L1288 only returns the BranchLastAttributedTouchData object.

Reply:
If you think the promise rejection is not an immediate/mandatory requirement for pushing this out, I'll be more than happy merge the PR. With that said, I would also be happy owning the native iOS change to support this and then make the RN change as well if so desired.

Reply:
I know we have a client actively waiting on this release, so if you're comfortable, I will merge the PR. I can also own making the change on our native iOS SDK to support both LATD and Error object returns within the completion and update the RN SDK to handle said change for a future release.